### DISCUSSION TITLE
Deployed to Heroku

### DISCUSSION THREAD

**Discussion by sgwcollins** [mirumee/saleor-dashboard#2247] [General]
I followed these instructions!
https://docs.saleor.io/docs/3.x/developer/running-saleor/heroku

However whenever i refresh i get logged out did i miss a step?

**Comment by CossackDex** [comment]
Hi @sgwcollins 

did you follow our authentication guide which you can find here -> https://docs.saleor.io/docs/3.x/developer/users#authentication,

Have a great day!
